main.py

```diff
@@ -1,10 +1,60 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_json:
+        list_of_players = json.load(players_json)
+
+    for name, info in list_of_players.items():
+        info_race = info["race"]
+        list_of_info_skills = info["race"]["skills"]
+        info_guild = info["guild"]
+
+        race = info_race["name"]
+        guild = None
+
+        if not Race.objects.filter(name=race).exists():
+            race = Race(
+                name=race,
+                description=info_race["description"]
+            )
+            race.save()
+        else:
+            race = Race.objects.get(name=race)
+
```

Comment:
Though your code works correct, you can simplify it using get_or_create() function.